ecl/hql/hqlutil.cpp

```diff
@@ -8855,12 +8859,14 @@ bool ConstantRowCreator::processFieldValue(IHqlExpression * optLhs, ITypeInfo *
     case type_groupedtable:
         {
             assertex(optLhs);
+            IHqlExpression * defaultAttrib = optLhs->queryAttribute(defaultAtom);
             IHqlExpression * field = optLhs->queryChild(1);
-            if (!field->hasAttribute(countAtom) && !field->hasAttribute(sizeofAtom))
+            if (defaultAttrib || (!field->hasAttribute(countAtom) && !field->hasAttribute(sizeofAtom)))
```

Comment:
not sure why this || is here.  I don't think count or sizeof forms can be generated inline.